source/common/stats/thread_local_store.h

```diff
@@ -413,6 +414,7 @@ class ThreadLocalStoreImpl : Logger::Loggable<Logger::Id::stats>, public StoreRo
   std::vector<HistogramSharedPtr> deleted_histograms_;
 
   Thread::ThreadSynchronizer sync_;
+  std::atomic<uint64_t> next_scope_id_;
```

Comment:
Technically shouldn't this be zero initialized somewhere?

Reply:
Probably...I couldn't find a definitive answer but it certainly is the pattern in the file.